<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LookConfig</class>
 <widget class="QWidget" name="LookConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="qgbLayout">
     <property name="title">
      <string>Layout</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutLayout">
      <item row="1" column="0">
       <widget class="QRadioButton" name="qrbLClassic">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Classic</string>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="qrbLStacked">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Stacked</string>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="qlLStacked">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>skin:layout_stacked.svg</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="qlLHybrid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>skin:layout_hybrid.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="qrbLHybrid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Hybrid</string>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QRadioButton" name="qrbLCustom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="qlLCustom">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>skin:layout_custom.svg</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="qlLClassic">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="pixmap">
         <pixmap>skin:layout_classic.svg</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QGroupBox" name="qgbLookFeel">
     <property name="title">
      <string>Look and Feel</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="qliStyle">
        <property name="text">
         <string>Style</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="qcbStyle">
        <property name="toolTip">
         <string>Basic widget style</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;This sets the basic look and feel to use.&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="qliCSS">
        <property name="text">
         <string>Skin</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="qleCSS">
        <property name="toolTip">
         <string>Skin file to use</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;This sets which skin Mumble should use.&lt;/b&gt;&lt;br /&gt;The skin is a style file applied on top of the basic widget style. If there are icons in the same directory as the style sheet, those will replace the default icons.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="qpbSkinFile">
        <property name="text">
         <string>&amp;Browse...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="qliLanguage">
        <property name="text">
         <string>Language</string>
        </property>
        <property name="buddy">
         <cstring>qcbLanguage</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QComboBox" name="qcbLanguage">
        <property name="toolTip">
         <string>Language to use (requires restart)</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;This sets which language Mumble should use.&lt;/b&gt;&lt;br /&gt;You have to restart Mumble to use the new language.</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="qcbHighContrast">
        <property name="toolTip">
         <string>Apply some high contrast optimizations for visually impaired users</string>
        </property>
        <property name="text">
         <string>Optimize for high contrast</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="qgbApplication">
     <property name="title">
      <string>Application</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0">
      <item row="0" column="0">
       <widget class="QLabel" name="qliAlwaysOnTop">
        <property name="text">
         <string>Always On Top</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="qcbAlwaysOnTop">
        <property name="toolTip">
         <string>This setting controls when the application will be always on top.</string>
        </property>
        <property name="whatsThis">
         <string>This setting controls in which situations the application will stay always on top. If you select &lt;i&gt;Never&lt;/i&gt; the application will not stay on top. &lt;i&gt;Always&lt;/i&gt; will always keep the application on top. &lt;i&gt;In minimal view&lt;/i&gt; / &lt;i&gt;In normal view&lt;/i&gt; will only keep the application always on top when minimal view is activated / deactivated.</string>
        </property>
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>In minimal view</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>In normal view</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="qcbShowContextMenuInMenuBar">
        <property name="toolTip">
         <string>Adds user and channel context menus into the menu bar</string>
        </property>
        <property name="text">
         <string>Show context menu in menu bar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="qcbAskOnQuit">
        <property name="toolTip">
         <string>Ask whether to close or minimize when quitting Mumble.</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;If set, will verify you want to quit if connected.&lt;/b&gt;</string>
        </property>
        <property name="text">
         <string>Ask on quit while connected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="qgbTray">
     <property name="title">
      <string>Tray Icon</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="qcbHideTray">
        <property name="toolTip">
         <string>Hide the main Mumble window in the tray when it is minimized.</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;If set, minimizing the Mumble main window will cause it to be hidden and accessible only from the tray. Otherwise, it will be minimized as a window normally would.&lt;/b&gt;</string>
        </property>
        <property name="text">
         <string>Hide in tray when minimized</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="qcbStateInTray">
        <property name="toolTip">
         <string>Displays talking status in system tray</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="text">
         <string>Show talking status in tray icon</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="qgbChannel">
     <property name="title">
      <string>Channel Tree</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
      <item row="0" column="0">
       <widget class="QLabel" name="qliChannelDrag">
        <property name="text">
         <string>Channel Dragging</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="qcbChannelDrag">
        <property name="toolTip">
         <string>This changes the behavior when moving channels.</string>
        </property>
        <property name="whatsThis">
         <string>This sets the behavior of channel drags; it can be used to prevent accidental dragging. &lt;i&gt;Move&lt;/i&gt; moves the channel without prompting. &lt;i&gt;Do Nothing&lt;/i&gt; does nothing and prints an error message. &lt;i&gt;Ask&lt;/i&gt; uses a message box to confirm if you really wanted to move the channel.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="qliExpand">
        <property name="text">
         <string>Expand</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="qcbExpand">
        <property name="toolTip">
         <string>When to automatically expand channels</string>
        </property>
        <property name="whatsThis">
         <string>This sets which channels to automatically expand. &lt;i&gt;None&lt;/i&gt; and &lt;i&gt;All&lt;/i&gt; will expand no or all channels, while &lt;i&gt;Only with users&lt;/i&gt; will expand and collapse channels as users join and leave them.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="qcbUsersTop">
        <property name="toolTip">
         <string>List users above subchannels (requires restart).</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;If set, users will be shown above subchannels in the channel view.&lt;/b&gt;&lt;br /&gt;A restart of Mumble is required to see the change.</string>
        </property>
        <property name="text">
         <string>Users above Channels</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="qcbShowUserCount">
        <property name="toolTip">
         <string>Show number of users in each channel</string>
        </property>
        <property name="text">
         <string>Show channel user count</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
